pages/home.js

```diff
@@ -1,12 +1,37 @@
 const { I } = inject();
+const Helper = require('../helpers/helper.js');
 
 module.exports = {
-myAccountSpoiler: { xpath: '//*[@id="top-links"]/ul/li/span/span' },
-registrerButton: { xpath: '//a[.="Register"]' },
+  myAccountSpoiler: { xpath: '//*[@id="top-links"]/ul/li/span/span' },
+  registrerButton: { xpath: '//a[.="Register"]' },
+  emptyCartText: { xpath: '//*[@id="cart"]/ul/li/p' },
+  cartIcon: { css: '#cart > button > i' },
+  trashIcon: { css: "i.linearicons-trash" },
+  addToCartButton: { xpath: '//*[@id="button-cart"]' },
+
+  openRegistrationPage() {
+    I.click(this.myAccountSpoiler);
+    I.click(this.registrerButton);
+  },
+
+  async checkCartIsEmpty() {
+    I.click(this.cartIcon);
+    return await Helper.checkElementIsVisible(this.emptyCartText);
+  },
+
+  async emptyCart() {
+    let isCartEmpty = await this.checkCartIsEmpty();
+    let amount = await I.grabAttributeFromAll({ css: "i.linearicons-trash" }, "class");
+    if (!isCartEmpty) {
+      for (i = 0; i < amount.length; i++) {
```

Comment:
I added 3 products to cart before test and it failed:
![image](https://user-images.githubusercontent.com/19595492/209718869-beeb501d-31f8-47cd-83d4-fefc1978a30c.png)
That happens because not all products were removed from cart by this cycle.
That could happen since framework does not wait for product to disappear and can place two clicks on same product.
I suggest to use specific locator in cycle for removeCartIcon, e.g. `'(//i[@class="linearicons-trash"])[3]'` for 3rd item in cart and remove products from last to first -> 3, 2, 1.